packages/langium/test/workspace/workspace-lock.test.ts

```diff
@@ -59,10 +59,10 @@ describe('WorkspaceLock', () => {
 
     test('Write action results can be awaited', async () => {
         const mutex = new DefaultWorkspaceLock();
-        const now = Date.now();
+        const now = performance.now();
         const magicalNumber = await mutex.read(() => new Promise(resolve => setTimeout(() => resolve(42), 10)));
         // Confirm that at least 10ms have elapsed
-        expect(Date.now() - now).toBeGreaterThanOrEqual(10);
+        expect(performance.now() - now).toBeGreaterThanOrEqual(10);
```

Comment:
The CI is failing here due to the higher precision:

![image](https://github.com/user-attachments/assets/05ccb132-a713-48be-a519-dd5a94bee65a)

We should decrease the value from 10 to something like 5 (which should be good enough).